src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String INFORMATION_SEPARATOR = ",";
```

Comment:
this constant can be renamed simple as `SEPARATOR` or 'FILE_SEPARATOR', word 'INFORMATION' has quite abstract meaning and doesn't add useful details.